pkg/kn/commands/source/container/create.go

```diff
@@ -76,14 +76,10 @@ func NewContainerCreateCommand(p *commands.KnParams) *cobra.Command {
 
 			b := v1.NewContainerSourceBuilder(name).Sink(*objectRef).PodSpec(*podSpec)
 			err = srcClient.CreateContainerSource(cmd.Context(), b.Build())
-			if err != nil {
-				return fmt.Errorf(
-					"cannot create ContainerSource '%s' in namespace '%s' "+
-						"because: %s", name, namespace, err)
```

Comment:
Does the error message still have the same wording even without this wrapper?

Reply:
no, made it consistent across sources like ping and API server. Also it was difficult to cover this portion as all possible errors are handled before this step, like sink resolution, pod spec errors etc.. If you think this adds value I can put it back.

Reply:
I was just curious if it's altering the messages significantly. Anyway, I'm glad to hear it's consistent now. :+1: